conf.d/rabbitmq.yaml.example

```diff
@@ -3,10 +3,13 @@ init_config:
 instances:
   # for every instance a 'rabbitmq_api_url' must be provided, pointing to the api
   # url of the RabbitMQ Managment Plugin (http://www.rabbitmq.com/management.html)
-  # optional: 'rabbitmq_user' (default: guest) and 'rabbitmq_pass' (default: guest)
+  # optional: 'rabbitmq_user' (default: guest), 'rabbitmq_pass' (default: guest),
+  # and 'tag_families' (default: false) to tag queue "families" based off of regex
+  # matching.
   - rabbitmq_api_url: http://localhost:15672/api/
     rabbitmq_user: guest
     rabbitmq_pass: guest
+  # tag_families: true
```

Comment:
nitpick: could you add 2 spaces of indentation here so that the commented-out `tag_queues` parameter is at the same level as the other commented-out instance-level parameters (e.g. `nodes`, queues`) :)